google-map.html

```diff
@@ -173,6 +228,21 @@
         });
         this.contentChanged();
         setupDragHandler_.bind(this)();
+
+        this._forwardEvent('click');
+        this._forwardEvent('dblclick');
+        this._forwardEvent('mousedown');
+        this._forwardEvent('mousemove');
+        this._forwardEvent('mouseout');
+        this._forwardEvent('mouseover');
+        this._forwardEvent('mouseup');
+        this._forwardEvent('rightclick');
+      },
+
+      _forwardEvent: function(name) {
+        google.maps.event.addListener(this.marker, name, function(event) {
+          this.fire('google-map-marker-' + name, event);
```

Comment:
Why not just fire the name of the event? Seems like it would be cleaner that way.

`this.fire(name, event);`